book/2 - Configuration Basics/1 - Configuration Overview.html

```diff
@@ -44,5 +44,5 @@
 <h4>Permissions</h4>
 
 <p>
-Setting up player permissions is most easily done via the web admin. You can also use the <code>rank &lt;playername&gt;</code> command from the server console.
+Permissions allow different players to access different commands and features. Each plugin has its own permissions. Setting up player permissions is most easily done via the web admin. You can also use the <code>rank &lt;playername&gt;</code> command from the server console. <a href="https://github.com/cuberite/Core">Click here</a> to see the command and permissions list for the default commands, which are provided by the Core plugin.
```

Comment:
Please no links with "click here" as the text. Be descriptive and remember this is supposed to be printable too :)